packages/image/__tests__/image.spec.ts

```diff
@@ -2,20 +2,17 @@ import { mount } from '@vue/test-utils'
 import Image from '../src/index.vue'
 import { nextTick } from 'vue'
 
-import { IMAGE_SUCCESS, IMAGE_FAIL } from '../../test-utils/mock'
+import { IMAGE_SUCCESS, IMAGE_FAIL, mockImageEvent } from '../../test-utils/mock'
 
+// firstly wait for image event
+// secondly wait for vue render
+async function doubleWait() {
+  await nextTick()
+  await nextTick()
+}
```

Comment:
I wonder under what circumstances do we need this double wait?

Reply:
Please read the comment above.